tests/common/jormungandr/starter.rs

```diff
@@ -76,53 +81,57 @@ fn is_node_up(output: Output) -> bool {
     }
 }
 
-pub fn start_jormungandr_node(mut config: &mut JormungandrConfig) -> ProcessKillGuard {
+pub fn start_jormungandr_node(mut config: &mut JormungandrConfig) -> JormungandrProcess {
```

Comment:
Why this mutability fest for `JormungandrConfig`? Should the function modify the configuration for later use?